ci/profileserver_linux.sh

```diff
@@ -0,0 +1,20 @@
+#! /bin/bash -vex
```

Comment:
how about calling those files `linux_*` and `win_*` so all platform specific files are prefixed?

Reply:
Sure. This was following the convention of our `sign_{mac,win}` scripts, so we should probably also change those too.

Reply:
yep, will do that